Added deeplink info (#231)
omise/omise-ios: README.md

> [![](https://cdn.omise.co/assets/omise.png)](https://omise.co)

Author: I am not sure where the CDN for Opn is. I tried cdn.opn.ooo and cd.docs.opn.ooo but they do not resolve. I will ask marketing where the logo is.

I have changed links to docs.opn.ooo, except for github - which is still github.com/omise/....

Also should support email change to support@opn.ooo?

> [![](https://cdn.omise.co/assets/omise.png)](https://omise.co)

Reviewer: > I am not sure where the CDN for Opn is. I tried cdn.opn.ooo and cd.docs.opn.ooo but they do not resolve. I will ask marketing where the logo is.

OK

> I have changed links to docs.opn.ooo, except for github - which is still github.com/omise/....

good

> Also should support email change to [support@opn.ooo](mailto:support@opn.ooo)?

yes

> [![](https://cdn.omise.co/assets/omise.png)](https://omise.co)

Author: Done.

> [![](https://cdn.omise.co/assets/omise.png)](https://omise.co)

Reviewer: In the interests of simplifying, I'm inclined to just remove the image to align with our other public repos.

# Opn Payments iOS SDK

[![Carthage compatible](https://img.shields.io/badge/Carthage-compatible-4BC51D.svg?style=flat-square)](https://github.com/Carthage/Carthage)
[![Swift Package Manager compatible](https://img.shields.io/badge/Swift%20Packager%20Manager-compatible-brightgreen?style=flat-square)](https://swift.org/package-manager)
[![](https://img.shields.io/badge/email-support-yellow.svg?style=flat-square)](mailto:support@omise.co)
![CI](https://github.com/omise/omise-ios/workflows/CI/badge.svg?branch=master)

> [Opn Payments](https://www.omise.co/) is a payment service provider operating

Reviewer: ```suggestion
[Opn Payments](https://docs.opn.ooo/) is a payment service provider operating
```

in Thailand, Japan, and Singapore. Opn Payments provides a set of APIs that
help merchants of any size accept payments online.

The Opn Payments iOS SDK provides bindings for

> [tokenizing credit cards](https://www..co/tokens-api) and

Reviewer: ```suggestion
[tokenizing credit cards](https://docs.opn.ooo/tokens-api) and
```

> [![](https://cdn.omise.co/assets/omise.png)](https://opn.ooo)

Reviewer: ```suggestion
```

### Authorizing Payment

> Some payment methods require the customers to authorize the payment via an authorized URL. This includes the [3-D Secure verification](https://www.omise.co/fraud-protection#3-d-secure), [Internet Banking payment](https://www.omise.co/offsite-payment), [Alipay](https://www.omise.co/alipay). The Opn Payments iOS SDK provides a built-in class to do the authorization.

Reviewer: ```suggestion
Some payment methods require the customers to authorize the payment via an authorized URL. This includes the [3-D Secure verification](https://docs.opn.ooo/fraud-protection#3-d-secure), [Internet Banking payment](https://docs.opn.ooo/offsite-payment), [Alipay](https://docs.opn.ooo/alipay). The Opn Payments iOS SDK provides a built-in class to do the authorization.
```

## LICENSE

> MIT [See the full license text](https://github.com//-ios/blob/master/LICENSE)

Reviewer: ```suggestion
MIT [See the full license text](https://github.com/omise/omise-ios/blob/master/LICENSE)
```
